src/PrimaryAuthController.js

```diff
@@ -18,7 +18,7 @@ define([
   'models/PrimaryAuth',
   'views/shared/Footer',
   'util/BaseLoginController',
-  'util/DeviceFingerprint'
+  'util/DeviceFingerprint',
```

Comment:
we may not need comma here

Reply:
Not a problem. We add trailing commas in these cases so when we add the next item we dont forget to add the comma. Linter is passing so no worries I guess.